ovs/dal/tests/test_alba.py

```diff
@@ -112,87 +112,124 @@ def test_localstack(self):
         ASDManagerClient.test_exceptions[node] = {}
         ASDManagerClient.test_results[node] = {'get_stack': {},
                                                'get_metadata': {'_version': 3}}
-        expected = {'node_1': {}}
+        expected = {}
         node.invalidate_dynamics()
-        self.assertDictEqual(alba_backend._local_stack(), expected)
+        self.assertDictEqual(node._stack(), expected)
 
         # Validate local_stack when the node is offline
         ASDManagerClient.test_exceptions[node] = {'get_stack': requests.ConnectionError('test'),
                                                   'get_metadata': requests.ConnectionError('test')}
-        ASDManagerClient.test_results[node] = {'get_stack': {'alba_disk_1': {'available': True,
-                                                                             'device': '/dev/disk/by-id/alba_disk_1',
-                                                                             'name': 'alba_disk_1',
-                                                                             'osds': {},
-                                                                             'node_id': node.node_id,
-                                                                             'state': 'ok'}},
-                                               'get_metadata': {'_version': 3}}
-        expected = {'node_1': {}}
+        ASDManagerClient.test_results[node] = {'get_stack': {},
+                                               'get_metadata': {}}
+        expected = {}
         node.invalidate_dynamics()
-        self.assertDictEqual(alba_backend._local_stack(), expected)
+        self.assertDictEqual(node._stack(), expected)
 
         # Validate uninitialized disks
+        def _move(info):
```

Comment:
Why implement this piece of code? Can't you just write the expacted values right away being 'status' and 'status_detail'
Instead of wrting 'state' in expected and 'move' it?